command/volume_status_csi.go

```diff
@@ -2,11 +2,13 @@ package command
 
 import (
 	"fmt"
+	"io"
 	"sort"
 	"strings"
 
 	"github.com/hashicorp/nomad/api"
 	"github.com/hashicorp/nomad/nomad/structs"
+	"github.com/pkg/errors"
```

Comment:
seems like the stdlib `errors` should cover what's used here
https://golang.org/pkg/errors/#example_Is

Reply:
`goimports` getting overzealous on me here. Thanks for catching that.